code/index.html

```diff
@@ -2,21 +2,79 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+       <!-- Some kind of logo and navbar -->
+    <header class="header-container">
+      <h1 class="top-title">Happy News</h1>
+      <img class="logo" src="News-icon.png" alt="Logo">
+      <nav class="nav-container">
+        <a class="nav-link" href="About">About</a>
+        <a class="nav-link" href="Search">Search</a>
+        <a class="nav-link" href="contact">Contact</a>
+      </nav>
     </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+      
+    <!-- The big news content -->
+    <section class="big-news-container">
+      <article>
+       <h2 class="big-news-title">Politics</h2>
+        <div class="big-news-items">
+        <div class="img-big-news">
+          <img src="./USA.jpg" alt="Bild på en stor nyhet" width="500px">
+        </div>
+        <p class="big-news-text">"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+      </div>
+      </article>
     </section>
```

Comment:
Maybe you don't need both a `section` and an `article` tag to wrap this content? 😇

Reply:
Would it be wrong to keep it that way? If I remove one of them the structure  with the title above the picture and paragraph will change, or is it better to remove one of them and make one more div instead to be able to place the items inside the container the way I want?